creator-node/src/index.ts

```diff
@@ -100,6 +101,14 @@ const startApp = async () => {
     )
   }
 
+  try {
+    const solSigningAuthority = Keypair.fromSeed(privateKeyBuffer)
+    const solSigningAuthPubKey = solSigningAuthority.publicKey
+    config.set('solSigningAuthPubKey', `${solSigningAuthPubKey}`)
```

Comment:
we should define this config var as a property in config.js for the documentation even if it's not set as an env var

Reply:
to clarify, we don't actually want it to be configurable? cause this code block would override whatever the config/env var sets. if that's the case and it's not really configurable then maybe it would make more sense to set it on the app like Hareesh suggests?